src/app/chip_data_model.gni

```diff
@@ -137,8 +138,22 @@ template("chip_data_model") {
                                      [ invoker.zap_file ])
     }
 
+    _cust_impl_clusters = []
```

Comment:
```suggestion
    _custom_impl_clusters = []
```

Reply:
agreed, `cust` is too much abbreviated, `custom` is better.
I just changed all occurrences and updated the PR